src/redux/user/userActions.js

```diff
@@ -0,0 +1,3 @@
+import { setUser, resetUser } from './userReducer';
+
+export { setUser, resetUser };
```

Comment:
this may be redundant for redux toolkit.. lets remove userActions on second thought